es/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 
-&#10230;
+&#10230; Pérdida de cross-entropy - En el contexto de las redes neuronales, la pérdida de cross-entropy L(z,y) es utilizada comunmente y definida de la siguiente manera:
 
 <br>
 
 **11. Learning rate ― The learning rate, often noted α or sometimes η, indicates at which pace the weights get updated. This can be fixed or adaptively changed. The current most popular method is called Adam, which is a method that adapts the learning rate.**
 
-&#10230;
+&#10230; Velocidad de aprendizaje - La velocidad de aprendizaje, denotada como α o algunas veces η, indica a que ritmo los pesos son actualizados. Este valor puede ser fijo o cambiar de forma adaptativa. El método más popular en este momento es llamado Adam, que es un método que se adapta a la velocidad de aprendizaje.
 
 <br>
 
 **12. Backpropagation ― Backpropagation is a method to update the weights in the neural network by taking into account the actual output and the desired output. The derivative with respect to weight w is computed using chain rule and is of the following form:**
 
-&#10230;
+&#10230; Retropropagación - La retropropagación, o propagación inversa, es un método de actualización de los pesos en una red neuronal, tomando en cuenta la salida actual y la salida esperada. La derivada respecto al peso w es computada utilizando la regla de la cadena y se expresa de la siguiente forma:
 
 <br>
 
 **13. As a result, the weight is updated as follows:**
 
-&#10230;
+&#10230; Como resultado, el peso es actualizado de la siguiente forma:
 
 <br>
 
 **14. Updating weights ― In a neural network, weights are updated as follows:**
 
-&#10230;
+&#10230; Actualizando pesos - En una red neuronal, los pesos son actualizados de la siguiente forma:
 
 <br>
 
 **15. Step 1: Take a batch of training data.**
 
-&#10230;
+&#10230; Paso 1: Tomar un grupo de los datos de entrenamiento.
 
 <br>
 
 **16. Step 2: Perform forward propagation to obtain the corresponding loss.**
 
-&#10230;
+&#10230; Paso 2: Realizar propagación hacia adelante para obtener la pérdida correspondiente.
 
 <br>
 
 **17. Step 3: Backpropagate the loss to get the gradients.**
 
-&#10230;
+&#10230; Paso 3: Propagar inversamente la pérdida para obtener los gradientes.
 
 <br>
 
 **18. Step 4: Use the gradients to update the weights of the network.**
 
-&#10230;
+&#10230; Paso 4: Utiliza los gradientes para actualizar los pesos de la red.
 
 <br>
 
 **19. Dropout ― Dropout is a technique meant at preventing overfitting the training data by dropping out units in a neural network. In practice, neurons are either dropped with probability p or kept with probability 1−p**
 
-&#10230;
+&#10230;Abandono - El retiro es una técnica para prevenir el sobreajuste de los datos de aprendizaje descartando unidades en una red neuronal. En la práctica, las neuronas son retiradas con una probabilidad de p o se mantienen con una probabilidad de 1-p.
```

Comment:
You translated "Dropout" two times in two different ways. Translate it as "_[retiro](https://developers.google.com/machine-learning/crash-course/glossary?hl=es-419)_" as it appears in the Google glossary for ML.